ores/wsgi/util.py

```diff
@@ -82,3 +84,17 @@ def parse_injection(request, rev_id):
         return {rev_id: cache} if len(cache) > 0 else None
     except Exception as e:
         raise CacheParsingError(e)
+
+
+def nocache(route):
+    @wraps(route)
+    def no_cache(*args, **kwargs):
+        response = make_response(route(*args, **kwargs))
+        response.headers['Cache-Control'] = \
+            "no-store, no-cache, must-revalidate, post-check=0, " + \
```

Comment:
must-revalidate should be sent by servers IF AND ONLY IF failure to revalidate a request could result in incorrect operation (like a transaction failing), which is arguably not the case here.